### PR TITLE
feat: integrate Tamagui

### LINKED ISSUE #12: Properly integrate Tamagui
When trying to use Tamagui the project will not run, either because `@tamagui/vite-plugin` is trying to use React 18 if we use React 17 or the app crashes at runtime.

### PR DESCRIPTION
**Pull request by GoNZooo**
Original author: @RadoslavDimchev 

This should initialize Tamagui properly and allow us to create and use components based on it.

Closes #12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,24 +2,33 @@
   "name": "nimbus-gui",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "NODE_ENV=development TAMAGUI_TARGET=web vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
```
Comment (RadoslavDimchev) [inline package.json:9]: We should add TAMAGUI_TARGET=web in preview and storybook script

File: index.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <title>Vite + React + TS</title>
   </head>
   <body>
+    <script>window.global = window;</script>
```
Comment (RadoslavDimchev) [inline index.html:10]: We can remove this script tag

**Review by HristoNedelkov — APPROVED** [review]
I can't wait for you to merge it so I can start enjoying it :)

File: package.json (hunk 1 above)
Comment (GoNZooo) [reply package.json:9]: This should be done in ee25c43, but it turned out we could remove it in all/most cases instead.

File: index.html (hunk 2 above)
Comment (GoNZooo) [reply index.html:10]: Unfortunately we cannot; Tamagui requires this. :(

**Review by RadoslavDimchev — APPROVED** [review]

**Pull request merged**